README.md

```diff
@@ -81,10 +81,11 @@ The OpenTracing and OpenCensus projects have merged into a new CNCF project call
 
 ### Multiple storage backends
 
-Jaeger supports two popular open source NoSQL databases as trace storage backends: Cassandra and Elasticsearch.
-There is also embedded database support using [Badger](https://github.com/dgraph-io/badger).
-There are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB.
-Jaeger also ships with a simple in-memory storage for testing setups.
+Jaeger can be used with a growing a number of storage backends:
+* It natively supports two popular open source NoSQL databases as trace storage backends: Cassandra and Elasticsearch.
+* It integrates via a gRPC API with other well known databases that have been certified to be Jaeger compliant: [TimescaleDB via Promscale](https://github.com/timescale/promscale), [ClickHouse](https://github.com/jaegertracing/jaeger-clickhouse).
```

Comment:
```suggestion
* It integrates via a gRPC API with other well known databases that have been certified to be Jaeger compliant: [PostgreSQL with Promscale](https://github.com/timescale/promscale), [ClickHouse](https://github.com/jaegertracing/jaeger-clickhouse).
```

Reply:
@ramonguiu Unfortunately the upstream PR got merged without this change. I will try to piggyback it with some other changes. Thanks.

Reply:
https://github.com/jaegertracing/jaeger/pull/3985